main.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Could you please add a comment on how to deploy this function?
I mean the command you would use to deploy this function and set the env variable.

Reply:
Also, please rename main.py to cloud_geolocalizer.py.

Reply:
Add some comments on the top of main module. https://cloud.google.com/functions/docs/writing/, however, the main.py is the required standard name for Google Cloud Functions